### DISCUSSION TITLE
[Feature] Disable thumbnail generation completely

### DISCUSSION THREAD

**Discussion by baek-sang** [immich-app/immich#9048] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

No thumbnail generation is required.
It is required as an option.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Thumbnail generation is kind of essential to Immich working smoothly. The web gallery would be unusably slow without it, and features like smart search and such also depend on it.
